draft-ietf-quic-http.md

```diff
@@ -526,6 +526,16 @@ A frame includes the following fields:
 
 ## Frame Definitions {#frames}
 
+Frame types of the format `0xb + (0x1f * N)` are reserved to exercise the
+requirement that unknown types be ignored. These frames have no semantic
+meaning, and SHOULD be sent when application-layer padding is desired.  They MAY
```

Comment:
Holdover from the HTTP draft, since that's recommending this in preference to padding fields on HEADERS/DATA.  I agree, the normative isn't needed here.